### DISCUSSION TITLE
Focusrite Scarlett Solo - Listen only to right Channel on Jamulus. Ok with headphones (Stereo is just fine).

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#2176] [Technical Support & Help]
Hi:

I am using Focusrite Scarlett Solo + Cubase Pro + Ibanez Guitar.

I can only get right Chanel to be heard on Jamulus. On my headphones is ok (stereo).

What to do?

Thanks.

Great APP by the way.

**Comment by chrisrimple** [comment]
Scarlett Solo does not, when sending a signal via USB to a computer, separate left and right channels.  On Jamulus, set Audio = Mono and you should hear everything on both sides (left and right).  If you want more control, set Audio = Stereo, run two instances of Jamulus, and set Pan = 100% Left and Pan = 100% Right for them.

**Reply by ghost** [reply]
I tried today also with an M-Audio, same problem. Now I tried also with the input 1 (voice), and it works with Left Channel. Thanks

**Comment by dcorson-ticino-com** [comment]
As Chris said.
Also, if you are hearing left and right in your headphones and only one side is going to Jamulus, then you have direct monitoring turned on.  Using Jamulus you must always have direct monitoring turned off.  If you don't you will never really be able to synchronise with the others you are playing with.
Listen to what is coming back from the server, that is what the others are also hearing.

**Reply by ghost** [reply]
I tried today also with an M-Audio, same problem. Now I tried also with the input 1 (voice), and it works with Left Channel. Thanks

**Comment by ghost** [comment]
Hi:

I dont know what else I can do... I tried everything...

***@***.***

***@***.***

Thanks,

Lucio


Von: Chris Rimple ***@***.***>
Gesendet: Sonntag, 19. Dezember 2021 21:21
An: jamulussoftware/jamulus ***@***.***>
Cc: luciofernandes1972 ***@***.***>; Author ***@***.***>
Betreff: Re: [jamulussoftware/jamulus] Focusrite Scarlett Solo - Listen only to right Channel on Jamulus. Ok with headphones (Stereo is just fine). (Discussion #2176)


Scarlett Solo does not, when sending a signal via USB to a computer, separate left and right channels. On Jamulus, set Audio = Mono and you should hear everything on both sides (left and right). If you want more control, set Audio = Stereo, run two instances of Jamulus, and set Pan = 100% Left and Pan = 100% Right for them.

-
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjamulussoftware%2Fjamulus%2Fdiscussions%2F2176%23discussioncomment-1844076&data=04%7C01%7C%7C1ee02513641f48cf1b6408d9c32d157f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755420661405564%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=g0Al5lLAi3dxZlU3Juf%2FhR8M%2BazdPqPv3%2Fw1%2Bw4xv9A%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAW2EKXKJ44QSXCFL2C4RFFDURY5DBANCNFSM5KMFAZGQ&data=04%7C01%7C%7C1ee02513641f48cf1b6408d9c32d157f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755420661415521%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Zx9bJp6j9wWPgNm9QcGVRTf%2BGrYDIAFq2Aep1AtWL8A%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C1ee02513641f48cf1b6408d9c32d157f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755420661415521%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=FtpAj8ijC5KSqy6d1wxZpOmrJHhdE2ID2F4d0lzuaVg%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C1ee02513641f48cf1b6408d9c32d157f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755420661425470%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=zk5ZnmXKBhajq41AMUW6dMEo45AyJx%2F0pTzAWX0w%2F2s%3D&reserved=0>.

**Comment by ghost** [comment]
Hi:

Thanks. I have DM off.

I dont know what else I can do... I tried everything...

***@***.***

***@***.***

Thanks,

Lucio


Von: DonC ***@***.***>
Gesendet: Sonntag, 19. Dezember 2021 22:25
An: jamulussoftware/jamulus ***@***.***>
Cc: luciofernandes1972 ***@***.***>; Author ***@***.***>
Betreff: Re: [jamulussoftware/jamulus] Focusrite Scarlett Solo - Listen only to right Channel on Jamulus. Ok with headphones (Stereo is just fine). (Discussion #2176)


As Chris said.
Also, if you are hearing left and right in your headphones and only one side is going to Jamulus, then you have direct monitoring turned on. Using Jamulus you must always have direct monitoring turned off. If you don't you will never really be able to synchronise with the others you are playing with.
Listen to what is coming back from the server, that is what the others are also hearing.

-
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjamulussoftware%2Fjamulus%2Fdiscussions%2F2176%23discussioncomment-1844200&data=04%7C01%7C%7C879d9d1d2e5a473d9bfb08d9c33600c3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755458967905388%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=PEj0IXGqZjqIQUQ0L310ostjnHjKtqbWUIGlHCM0hhQ%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAW2EKXLSMVGT36YNFPGYEELURZESNANCNFSM5KMFAZGQ&data=04%7C01%7C%7C879d9d1d2e5a473d9bfb08d9c33600c3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755458967915345%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=5NbllG90XZbX4ArLrlceL%2Bn0R2YkYABR5YqpqmeWjpo%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C879d9d1d2e5a473d9bfb08d9c33600c3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755458967915345%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=NqDFhhe28LqMF%2BlGvRa0%2Bkvlk96wFrscP5f8u08bP5A%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C879d9d1d2e5a473d9bfb08d9c33600c3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637755458967925297%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=M0befBWROQw5SOdVfnDxv9RbsRBsjPD%2FaSYMgjcciAE%3D&reserved=0>.

**Comment by chrisrimple** [comment]
Do you have the Jamulus setting Audio Channels = Mono?

**Reply by ghost** [reply]
Yes. I tried today also with an M-Audio, same problem. Now I tried also with the input 1 (voice), and it works with Left Channel. Thanks

**Reply by ghost** [reply]
I will try this:

Points to note about inputs
If you want to connect 2 or more sources (e.g. voice + synth + guitar), it is important to note that Jamulus currently handles only 2 input channels (L/R). So the hardware being used must provide a mixed-down stereo output to Jamulus. Alternatively, run two instances of Jamulus, if your hardware supports it, or use virtual audio inputs to Jamulus for each hardware input.
Audio interfaces generally output a mixed signal on their analogue output, but separate signals (1 per source) on their digital output (USB/FireWire/Thunderbolt).
Mixers generally only output mixed-down signals on their analogue output.
Mixers with USB/FireWire/Thunderbolt generally output a mixed-down signal on their analogue output AND separate signals only (no mixed-down signal) on their digital output.
A few Mixers with USB/FireWire/Thunderbolt will either send only a mixed-down signal to the digital output (small/budget mixers), or else also ADD a stereo mixed-down signal to the separate signals on the digital output.
